### DISCUSSION TITLE
More body diversity/size inclusion

### DISCUSSION THREAD

**Discussion by danatodd** [HumanAIGC/OutfitAnyone#16] [Ideas]
This is very impressive tech to see, and it's exciting as a foundation for digital product development and retail ecommerce. But the demo is only showing a limited set of bodies. In the US, we have significant diversity in body shapes, sizes, and skin tones. If you can import more body meshes from scans, or photos, that would be most helpful for giving realistic virtual try on experiences, particularly for very curvy women or people of color. There are body scan databases that you can access or purchase from private sources like Alvanon, which account for different body ratios as well. There are a certain set of "morphotypes" that are recognizable body geometries/shapes. For example, even if two people are the same height and weight, their fat distribution and skeletal frame may create a very different body shap. 

About 50% of American women wear over size 14. The models in your demo would likely be a size 0 or XS in US standard sizing.

Additionally: I am wondering about its use as a true "try on" for body fitting. If you are working only in imagery and not technical pattern specs of the clothing items, it may mislead some people to believe that the item will fit their body. Is that a concern for you?

**Comment by malcolmroyceusa17** [comment]
I sell children's clothing, suits and dresses, and would love to be able to create an ai body of a given height and add the garments I sell to see fit, etc. And also show customers a more realistic view. Would help immensely with sizing for parents.

**Comment by corteiztracksuit11** [comment]
More Body Diversity and Size Inclusion: Corteiz Tracksuits Leading the Way
In today’s streetwear culture, style isn’t just about the fit—it’s about how it makes you feel. For too long, many fashion brands overlooked body diversity, leaving countless people without access to the latest styles in sizes that work for them. At [Corteiz](https://crtzde.com/), we’ve taken a different approach. Our tracksuits are designed to be inclusive, versatile, and empowering for every body type.
Designed for Every Shape and Size
The [Corteiz Tracksuit](https://crtzde.com/tracksuit/) isn’t made for a single body standard—it’s built for real people. With a wide range of sizes and adjustable fits, we make sure no one is left out of the culture. From streamlined cuts to relaxed oversized fits, every design is tailored to provide comfort and confidence for all.
Comfort Meets Culture
We know that streetwear is more than just clothing—it’s identity, movement, and self-expression. That’s why the Corteiz Tracksuit is crafted to support you wherever you are, whoever you are. Whether you’re into slim silhouettes or a looser, laid-back vibe, our tracksuits adapt to your personal style while celebrating inclusivity.
Authentic Streetwear for Everyone
Corteiz is redefining what it means to be part of the streetwear movement. Inclusivity isn’t an afterthought—it’s at the core of our vision. By offering tracksuits that look sharp, feel good, and fit a diverse community, we make sure everyone has the chance to step into culture with confidence.
Final Word
Body diversity and size inclusion matter. The Corteiz Tracksuit proves that streetwear can be fearless, stylish, and inclusive all at once. Because true street culture welcomes every body.
